skycultures/ticuna/description.en.utf8

```diff
@@ -0,0 +1,22 @@
+<h3 align="center"> The Astronomic Correlations</h3>
+<p align="justify">The astronomic correlations between Western and Ticuna constellations were identified by the Ticuna in November 2002 with the help of the astronomer Fernando Vieira (Planetarium Foundation of Rio de Janeiro City) during a study in the Science Center and Planetarium of the Universidade do Pará (CCPP) in the city of Belém   (Brazil), also using the Starry Night System. We are integrating in this investigation a Physics Professor at Pontifícia Universidade Católica de São Paulo (PUC-SP), and Astronomy in Culture researcher Walmir Thomazi Cardoso, who made some reinterpretations in a collaborative work with physics undergraduate student and Stellarium developer collaborator Youssif Ghanthous Filho, adapting and anchoring original drawings to the Stellarium. Nevertheless, every sky interpretation is contextual and circumstantial. Current collaborative investigations are looking for new astronomical observation together with the Ticuna, which will improve the initial findings.</p>
+<p align="justify">Correlating the Ticuna constellations with those of Western astronomy<cite id="cit2a"><a href="#cit2"><B>²</B></a></cite> , the form of Baweta (the Turtle Collective) is associated with the stars of the asterisms of the Pleiades and of Perseus. Its eyes are the feet of Perseus (Omicron Persei and Atik). The head of the V form that Western astronomy relates to Taurus is called Coyatchicüra (Cayman’s Jaw). In a Ticuna drawing, the Cayman’s left paw corresponds to the star Euratis and the right rear paw to ε (Epsilon), η (Eta) ζ (zeta). The tail is in ο (Omicrom) and ζ (Zeta) in Perseus. In turn, Wücütcha’s Leg is related to the stars that groups as Orion. The toes are Orion’s Belt and the bend of the legs are α (Alfa), γ (Gama),β (Beta), ε (Epsilon).</p>
+<p align="justify">Throughout the summer, the Anteater’s (Tchatü) Fight with the Jaguar (Ai) takes place<cite id="cit3a"><a href="#cit3"><B>³</B></a></cite> , the end of which coincides with the death and ascension of Wücütcha. Anteater’s eyes are in β in the Austral Triangle, his tail in the Austral Crown and body in the constellations of Norma and Ara. The Jaguar’s right paw is in α (Antares), σ (Sigma) and τ (Tau) in Scorpion; his eyes are in ε e μ, also in Scorpion and his left paw is in γ (Shaula) ν (nu) ι (iota) and θ (Theta) of the same constellation. According to the Ticuna, this was the result of the rape/murder of a pregnant woman. The victim’s relatives killed the beast associated with the Jaguar Clan to prevent the eclipse. Tchatü’s leaving the sky to the west at nightfall in the end of November coincides with the reappearance of Wücütcha’s Leg to the East, representing the beginning of the rainy season. Those correlations are illuminated by indigenous testimony reproduced bellow:</p>
```

Comment:
"This was the result... " What exactly? Generally, this section is hard to comprehend. What kind of eclipse? Is there a part of a story that has been skipped?